thewalrus/symplectic.py

```diff
@@ -89,6 +90,8 @@ def expand_vector(alpha, mode, N, hbar=2.0):
     Returns:
         array: phase-space displacement vector of size 2*N
     """
+    #alpharealdtype = (alpha.real).dtype
+    # This one is problematic because alpha is not a numpy object
```

Comment:
This should work if `alpha` is a complex number. Then `alpharealdtype` is `dtype('complex128)`.

```suggestion
    alpharealdtype = np.dtype(type(alpha))

```